helm_drydock/config.py

```diff
@@ -21,21 +21,13 @@
 
```

Comment:
Is there any concern that there is already a 'drydock' in PyPi? I was attempting to avoid a name conflict.